components.js

```diff
@@ -157,4 +157,100 @@ export const Section = window.components.Section || NotCompatible('5.0');
  *   );
  * }
  */
-export const ToggleSwitch = window.components.ToggleSwitch || NotCompatible('5.0');
\ No newline at end of file
+export const ToggleSwitch = window.components.ToggleSwitch || NotCompatible('5.0');
+
+ /**
+ * TextInput component.
+ *
+ * @type {import('react').ComponentType<{ hint: string, label: string, fieldError: string, multiline: boolean, description: string }>}
+ *
+ * @example
+ * 
+ * import { ToggleSwitch } from 'camunda-modeler-plugin-helpers/components';
+ *
+ * function CustomToggle(props) {
+ *   return (
+ *    <Formik initialValues={ initialValues } onSubmit={ this.onSubmit }>
+ *      {() => (
+ *        <Form>
+ *          <Field
+ *            component={ ToggleSwitch }
+ *            switcherLabel="Switcher label"
+ *            id={ id }
+ *            name={ name }
+ *            description="Toggle description"
+ *          />
+ *        </Form>
+ *       )}
+ *    </Formik>
+ *   );
+ * }
```

Comment:
This needs to be updated.

Reply:
```suggestion
 * @example
 * 
 * import { TextInput } from 'camunda-modeler-plugin-helpers/components';
 *
 * function CustomInput(props) {
 *   return (
 *    <Formik initialValues={ initialValues } onSubmit={ this.onSubmit }>
 *      {() => (
 *        <Form>
 *          <Field
 *            component={ TextInput }
 *            label="My input"
 *            id={ id }
 *.           multiline={false}
 *            name={ name }
 *            description="Custom description"
 *          />
 *        </Form>
 *       )}
 *    </Formik>
 *   );
 * }
 ```

Reply:
Added in [df906e7](https://github.com/camunda/camunda-modeler-plugin-helpers/pull/26/commits/df906e74e56971c8b76adf359cb945316371cb21)